internal/driver/mobile/app/darwin_ios.go

```diff
@@ -225,48 +230,12 @@ func drawloop() {
 		case <-theApp.publish:
 			theApp.publishResult <- PublishResult{}
 			return
-		case <-time.After(100 * time.Millisecond): // incase the method blocked!!
+		case <-time.After(defaultFramerate):
```

Comment:
Are you sure that is how the delay works? I thought that how fast the screen refreshed was down to when the `theApp.publish` channel is sent? Decreasing the time in this segment breaks the select more often, but it does not do anything extra when that happens...?

I think more relevant is the `Run()` in the main mobile driver which has the following code:
```
draw := time.NewTicker(time.Second / 60)
```

Reply:
> I thought that how fast the screen refreshed was down to when the theApp.publish channel is sent?

No. drawloop is executed on the system thread called by `drawInRect` (see drawin_ios.m), which executes all GL draw calls in DoWork. The `publish` channel is sent for calling glFlush, which will still be processed in this function (thread).

If you open the temporary generated Xcode file, running the application, debug panel shows you an Xcode measured fps number, which is literally measuring how fast the `drawInRect` returns.

Reply:
```
draw := time.NewTicker(time.Second / 60)
```

Only controls how regular the draw calls are emitted from the (pesudo) drawing thread. Note that these GL calls are actually send via a channel that actually executed in the `DoWork()`

Reply:
But don't we need to update both?
If the GL is updated to allow 120fps but we only send 60 times then we won't see the improvement will we?

Reply:
> But don't we need to update both?

Not. The actual FPS only depends on how fast the drawInRect can return, irrelevant to how fast the run produces draw calls.

If each tick of draw in Run produces much more draw calls, the drawInRect can still return soon (which is vita for responsiveness); If each tick of draw in Run produces a little draw call, that's already what we want (drawInRect has less work to process and returns fast).

Reply:
If draw is called at 60FPS, how many difference frames will be drawn into your 120FPS output?
Surely it is only 60? animations will slice their timeline to 60 pieces and we won't see the benefit of 120 FPS output.

Reply:
Done.